src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Invitations/InvitationsControllerNegativeTests.cs

```diff
@@ -289,6 +296,37 @@ public async Task CreateInvitation_AsPlanner_ShouldReturn_SpecificError_WhenMail
             );
         }
 
+        [TestMethod]
+        public async Task CreateInvitation_AsPlanner_ShouldUse_Fallback_WhenPlannerProblem()
```

Comment:
This test don't belong here: It should be an unit test in CreateInvitationCommandHandlerTests

Reply:
Since the fallback solution is an expected behavior you mean? Should still be an integration test? In that case placed in InvitationsControllerTests.cs.

Reply:
Already handled through HandleCreateInvitationCommand_ShouldCreateICSIfFusionApiFails in CreateInvitationCommandHandlerTests. Hence removing CreateInvitation_AsPlanner_ShouldUse_Fallback_WhenPlannerProblem.